ucp-kube-proxy
2018-06-22T15:27:21.907265723Z W0622 15:27:21.906438       1 server.go:191] WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.
2018-06-22T15:27:21.914833668Z time="2018-06-22T15:27:21Z" level=warning msg="Running modprobe ip_vs failed with message: `modprobe: ERROR: ../libkmod/libkmod.c:586 kmod_search_moddep() could not open moddep file '/lib/modules/3.10.0-693.11.6.el7.x86_64/modules.dep.bin'\nmodprobe: WARNING: Module ip_vs not found in directory /lib/modules/3.10.0-693.11.6.el7.x86_64`, error: exit status 1"
2018-06-22T15:27:21.940011402Z W0622 15:27:21.939188       1 server_others.go:268] Flag proxy-mode="" unknown, assuming iptables proxy
2018-06-22T15:27:21.943358043Z I0622 15:27:21.941788       1 server_others.go:122] Using iptables Proxier.
2018-06-22T15:27:31.943260817Z W0622 15:27:31.943081       1 server.go:580] Failed to retrieve node info: Get https://proxy.local:6444/api/v1/nodes/ip-172-31-18-229.ec2.internal: net/http: TLS handshake timeout
2018-06-22T15:27:31.944091737Z W0622 15:27:31.943213       1 proxier.go:471] invalid nodeIP, initializing kube-proxy with 127.0.0.1 as nodeIP
2018-06-22T15:27:31.944107397Z I0622 15:27:31.943350       1 server_others.go:157] Tearing down inactive rules.
2018-06-22T15:27:31.978838236Z E0622 15:27:31.978698       1 proxier.go:699] Failed to execute iptables-restore for nat: exit status 1 (iptables-restore: line 7 failed
2018-06-22T15:27:31.978853077Z )
2018-06-22T15:27:31.983145978Z I0622 15:27:31.982627       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_max' to 131072
2018-06-22T15:27:31.983158462Z I0622 15:27:31.982670       1 conntrack.go:52] Setting nf_conntrack_max to 131072
2018-06-22T15:27:31.983163153Z I0622 15:27:31.982703       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_established' to 86400
2018-06-22T15:27:31.983166915Z I0622 15:27:31.982726       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_close_wait' to 3600
2018-06-22T15:27:31.983170528Z I0622 15:27:31.982951       1 config.go:102] Starting endpoints config controller
2018-06-22T15:27:31.983173842Z I0622 15:27:31.982968       1 controller_utils.go:1041] Waiting for caches to sync for endpoints config controller
2018-06-22T15:27:31.983177231Z I0622 15:27:31.982992       1 config.go:202] Starting service config controller
2018-06-22T15:27:31.983180430Z I0622 15:27:31.982996       1 controller_utils.go:1041] Waiting for caches to sync for service config controller
2018-06-22T15:27:41.983993581Z E0622 15:27:41.983714       1 event.go:209] Unable to write event: 'Post https://proxy.local:6444/api/v1/namespaces/default/events: net/http: TLS handshake timeout' (may retry after sleeping)
2018-06-22T15:27:41.984687306Z E0622 15:27:41.984408       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-22T15:27:41.984704951Z E0622 15:27:41.984416       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-22T15:27:52.985681085Z E0622 15:27:52.985390       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-22T15:27:52.986500703Z E0622 15:27:52.986368       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-22T15:27:54.083330525Z I0622 15:27:54.083157       1 controller_utils.go:1048] Caches are synced for endpoints config controller
2018-06-22T15:27:54.083354242Z I0622 15:27:54.083225       1 controller_utils.go:1048] Caches are synced for service config controller
2018-06-28T16:42:14.571246965Z E0628 16:42:14.570862       1 reflector.go:315] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to watch *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=98195&timeoutSeconds=368&watch=true: net/http: TLS handshake timeout
2018-06-28T16:42:14.571269149Z E0628 16:42:14.570863       1 reflector.go:315] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to watch *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=835235&timeoutSeconds=528&watch=true: net/http: TLS handshake timeout
2018-06-28T16:42:25.572263394Z E0628 16:42:25.571953       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:42:25.573082544Z E0628 16:42:25.572806       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:42:36.573418566Z E0628 16:42:36.572997       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:42:36.574231028Z E0628 16:42:36.573917       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:42:47.574454784Z E0628 16:42:47.574031       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:42:47.574960786Z E0628 16:42:47.574851       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:42:58.575018707Z E0628 16:42:58.574913       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:42:58.576074193Z E0628 16:42:58.575782       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:43:09.576517621Z E0628 16:43:09.576169       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:43:09.576892770Z E0628 16:43:09.576657       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:43:20.577170002Z E0628 16:43:20.576933       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:43:20.578384756Z E0628 16:43:20.577920       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:43:31.578496046Z E0628 16:43:31.578023       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:43:31.578983237Z E0628 16:43:31.578680       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:43:42.579298659Z E0628 16:43:42.578942       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:43:42.579952829Z E0628 16:43:42.579665       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:56:41.530952340Z E0628 16:56:41.530617       1 reflector.go:315] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to watch *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=836465&timeoutSeconds=545&watch=true: net/http: TLS handshake timeout
2018-06-28T16:56:41.530981876Z E0628 16:56:41.530647       1 reflector.go:315] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to watch *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=835235&timeoutSeconds=530&watch=true: net/http: TLS handshake timeout
2018-06-28T16:56:52.531892031Z E0628 16:56:52.531644       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:56:52.532760625Z E0628 16:56:52.532426       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:57:03.533043680Z E0628 16:57:03.532741       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T16:57:03.533586212Z E0628 16:57:03.533471       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: net/http: TLS handshake timeout
2018-06-28T17:03:44.803360057Z W0628 17:03:44.801302       1 server.go:191] WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.
2018-06-28T17:03:44.810287108Z time="2018-06-28T17:03:44Z" level=warning msg="Running modprobe ip_vs failed with message: `modprobe: ERROR: ../libkmod/libkmod.c:586 kmod_search_moddep() could not open moddep file '/lib/modules/3.10.0-693.11.6.el7.x86_64/modules.dep.bin'\nmodprobe: WARNING: Module ip_vs not found in directory /lib/modules/3.10.0-693.11.6.el7.x86_64`, error: exit status 1"
2018-06-28T17:03:44.832175067Z W0628 17:03:44.831685       1 server_others.go:268] Flag proxy-mode="" unknown, assuming iptables proxy
2018-06-28T17:03:44.834530377Z I0628 17:03:44.833384       1 server_others.go:122] Using iptables Proxier.
2018-06-28T17:03:44.836504343Z W0628 17:03:44.834677       1 server.go:580] Failed to retrieve node info: Get https://proxy.local:6444/api/v1/nodes/ip-172-31-18-229.ec2.internal: dial tcp 172.31.18.229:6444: getsockopt: connection refused
2018-06-28T17:03:44.836516995Z W0628 17:03:44.834746       1 proxier.go:471] invalid nodeIP, initializing kube-proxy with 127.0.0.1 as nodeIP
2018-06-28T17:03:44.836521528Z I0628 17:03:44.834813       1 server_others.go:157] Tearing down inactive rules.
2018-06-28T17:03:44.915366792Z E0628 17:03:44.907533       1 proxier.go:699] Failed to execute iptables-restore for nat: exit status 1 (iptables-restore: line 7 failed
2018-06-28T17:03:44.915386753Z )
2018-06-28T17:03:44.915391141Z I0628 17:03:44.911562       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_max' to 131072
2018-06-28T17:03:44.915395231Z I0628 17:03:44.911597       1 conntrack.go:52] Setting nf_conntrack_max to 131072
2018-06-28T17:03:44.915410513Z I0628 17:03:44.911683       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_established' to 86400
2018-06-28T17:03:44.915415351Z I0628 17:03:44.911701       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_close_wait' to 3600
2018-06-28T17:03:44.915419168Z I0628 17:03:44.912369       1 config.go:202] Starting service config controller
2018-06-28T17:03:44.915423073Z I0628 17:03:44.912384       1 controller_utils.go:1041] Waiting for caches to sync for service config controller
2018-06-28T17:03:44.915426845Z I0628 17:03:44.912420       1 config.go:102] Starting endpoints config controller
2018-06-28T17:03:44.915438880Z I0628 17:03:44.912424       1 controller_utils.go:1041] Waiting for caches to sync for endpoints config controller
2018-06-28T17:03:44.922291381Z E0628 17:03:44.919331       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Endpoints: Get https://proxy.local:6444/api/v1/endpoints?resourceVersion=0: dial tcp 172.31.18.229:6444: getsockopt: connection refused
2018-06-28T17:03:44.922304821Z E0628 17:03:44.919420       1 event.go:209] Unable to write event: 'Post https://proxy.local:6444/api/v1/namespaces/default/events: dial tcp 172.31.18.229:6444: getsockopt: connection refused' (may retry after sleeping)
2018-06-28T17:03:44.925279304Z E0628 17:03:44.924059       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:73: Failed to list *api.Service: Get https://proxy.local:6444/api/v1/services?resourceVersion=0: dial tcp 172.31.18.229:6444: getsockopt: connection refused
2018-06-28T17:03:46.012670574Z I0628 17:03:46.012566       1 controller_utils.go:1048] Caches are synced for endpoints config controller
2018-06-28T17:03:46.012747275Z I0628 17:03:46.012575       1 controller_utils.go:1048] Caches are synced for service config controller
2018-06-28T17:27:14.463237207Z W0628 17:27:14.459453       1 server.go:191] WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.
2018-06-28T17:27:14.468478653Z time="2018-06-28T17:27:14Z" level=warning msg="Running modprobe ip_vs failed with message: `modprobe: ERROR: ../libkmod/libkmod.c:586 kmod_search_moddep() could not open moddep file '/lib/modules/3.10.0-693.11.6.el7.x86_64/modules.dep.bin'\nmodprobe: WARNING: Module ip_vs not found in directory /lib/modules/3.10.0-693.11.6.el7.x86_64`, error: exit status 1"
2018-06-28T17:27:14.521589046Z W0628 17:27:14.516520       1 server_others.go:268] Flag proxy-mode="" unknown, assuming iptables proxy
2018-06-28T17:27:14.527514498Z I0628 17:27:14.522315       1 server_others.go:122] Using iptables Proxier.
2018-06-28T17:27:14.594109191Z I0628 17:27:14.585078       1 server_others.go:157] Tearing down inactive rules.
2018-06-28T17:27:14.660768257Z E0628 17:27:14.658545       1 proxier.go:699] Failed to execute iptables-restore for nat: exit status 1 (iptables-restore: line 7 failed
2018-06-28T17:27:14.660791121Z )
2018-06-28T17:27:14.663910259Z I0628 17:27:14.662671       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_max' to 131072
2018-06-28T17:27:14.663923161Z I0628 17:27:14.662710       1 conntrack.go:52] Setting nf_conntrack_max to 131072
2018-06-28T17:27:14.663927343Z I0628 17:27:14.662739       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_established' to 86400
2018-06-28T17:27:14.663930979Z I0628 17:27:14.662755       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_close_wait' to 3600
2018-06-28T17:27:14.663934703Z I0628 17:27:14.663277       1 config.go:102] Starting endpoints config controller
2018-06-28T17:27:14.663938090Z I0628 17:27:14.663294       1 controller_utils.go:1041] Waiting for caches to sync for endpoints config controller
2018-06-28T17:27:14.663941678Z I0628 17:27:14.663319       1 config.go:202] Starting service config controller
2018-06-28T17:27:14.663952566Z I0628 17:27:14.663324       1 controller_utils.go:1041] Waiting for caches to sync for service config controller
2018-06-28T17:27:14.770270707Z I0628 17:27:14.765760       1 controller_utils.go:1048] Caches are synced for endpoints config controller
2018-06-28T17:27:14.770291515Z I0628 17:27:14.765961       1 controller_utils.go:1048] Caches are synced for service config controller
